applications/asset_tracker/prj.conf

```diff
@@ -39,11 +39,12 @@ CONFIG_NRF_MODEM_LIB_TRACE_ENABLED=n
 CONFIG_CLOUD_API=y
 CONFIG_NRF_CLOUD=y
 CONFIG_NRF_CLOUD_LOG_LEVEL_DBG=y
-CONFIG_NRF_CLOUD_AGPS=y
+CONFIG_NRF_CLOUD_AGPS=n
```

Comment:
Are these asset tracker cfg changes needed for this pr?

Reply:
oops -- no, mistake